packages/flutter/lib/src/physics/friction_simulation.dart

```diff
@@ -10,6 +10,20 @@ import 'simulation.dart';
 
 export 'tolerance.dart' show Tolerance;
 
+double _newtonsMethod({
+  required double guess,
+  required double target,
+  required double Function(double) f,
+  required double Function(double) df,
+  required int iterations
+}) {
+  double g = guess;
+  for (int i = 0; i < iterations; i++) {
+    g = g - (f(g) - target) / df(g);
+  }
+  return g;
```

Comment:
The style guide discourages abbreviations https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abbreviations

Can you expand this shorthand so it is easier to understand?